spec/factories/rating_factor_sets.rb

```diff
@@ -0,0 +1,21 @@
+FactoryGirl.define do
+  factory :rating_factor_set do
+    # association :person, ssn: '123456789', dob: "1/1/1965", gender: "female", first_name: "Sarah", last_name: "Smile"
+    association :carrier_profile
+
+    active_year { TimeKeeper.date_of_record.year }
+    max_integer_factor_key nil
+    default_factor_value 1.0
+    rating_factor_entries [ FactoryGirl.build(:rating_factor_entry) ]
+  end
```

Comment:
each rating_factor_set has many rating_factor_entries and is associated with a carrier profile. This is how we define what the group sizes are and their factors